packages/shared/components/popups/RevotePopup.svelte

```diff
@@ -30,12 +26,7 @@
     <Text type={TextType.h4} classes="mb-3">{localize('popups.revote.title')}</Text>
     <Text type={TextType.p} secondary>{localize('popups.revote.body')}</Text>
```

Comment:
![image](https://user-images.githubusercontent.com/105642810/215748648-6d28608f-d1f7-46b3-a699-979d08a15d67.png)

could you update the color of the body such that it matches the others?